core/commands/config.go

```diff
@@ -512,7 +512,14 @@ func editConfig(filename string) error {
 		return errors.New("ENV variable $EDITOR not set")
 	}
 
-	cmd := exec.Command(editor, filename)
+	editorAndArgs, err := shlex.Split(editor, true)
```

Comment:
Note: go-shlex has not had a commit in 5 years. However, the code of `go-shlex` is 193 lines long with some sane tests. Perhaps no changes are needed because it works perfectly?

I didn't see any Windows-specific tests.